pages/linux/gbp.md

```diff
@@ -0,0 +1,24 @@
+# gbp
+
+> A system to integrate the Debian package build system with Git.
+> More information: <http://honk.sigxcpu.org/projects/git-buildpackage/manual-html/gbp.html>.
+
+- Convert a pre-existing Debian package to gbp:
+
+`gbp import-dsc {{software_0.1-2.dsc}}`
+
+- Build a package using the default builder (`debuild`):
+
+`gbp buildpackage -jauto -us -uc`
```

Comment:
I can't find all those options in the [man page](http://honk.sigxcpu.org/projects/git-buildpackage/manual-html/man.gbp.buildpackage.html)

Reply:
Those are `OPTION_PASSED_TO_BUILD_CMD`. I know it's convoluted (hence the need for a tldr page!)